yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -343,6 +343,9 @@ func (yb *TargetYugabyteDB) CreateVoyagerSchema() error {
 			rows_imported BIGINT,
 			PRIMARY KEY (migration_uuid, data_file_name, batch_number, schema_name, table_name)
 		);`, BATCH_METADATA_TABLE_NAME),
+		fmt.Sprintf(`ALTER TABLE %s ALTER COLUMN data_file_name TYPE TEXT;`, BATCH_METADATA_TABLE_NAME),
```

Comment:
Should we do it for PG fall-forward/back scenario
And for Oracle, though we don't those creation steps in code but for docs as well https://docs.yugabyte.com/preview/yugabyte-voyager/migrate/live-fall-forward/#prepare-source-replica-database

Reply:
Completely forgot about source-replica, thanks :) 
Made the change for PG. Noted that we need to change docs for oracle. 
Remind me- why don't we do this automatically in the case of oracle?